api/src/test/java/org/openmrs/api/ConceptServiceTest.java

```diff
@@ -2887,6 +2891,9 @@ public void saveConcept_shouldSetAuditInfoIfAnItemIsRemovedFromAnyOfItsChildColl
 		assertNotNull(concept.getDateChanged());
 		Date date=concept.getDateChanged();
 
+		// A short delay to ensure a noticeable difference in dates
```

Comment:
Can't we achieve this without sleeping?

Reply:
I was trying to fix this brittle test but I will remove this sleep and solve this in a different task.